Tasks/AzureIoTEdgeV2/constant.ts

```diff
@@ -11,7 +11,7 @@ export default class Constants {
   public static iotedgehubdev = "iotedgehubdev";
   public static iotedgedevLockVersionKey = "IOTEDGEDEV_VERSION";
   public static iotedgehubdevLockVersionKey = "IOTEDGEHUBDEV_VERSION";
-  public static iotedgedevDefaultVersion = "3.0.0";
+  public static iotedgedevDefaultVersion = "3.2.0";
```

Comment:
>2 [](http://example.com/codeflow?start=46&length=1)

if we are upgrading anyway, should we pull in 3.3.0?

Reply:
Good point, I can make that change now